pkg/sdk/views_validations_gen.go

```diff
@@ -31,21 +29,7 @@ func (opts *CreateViewOptions) validate() error {
 	}
 	if valueSet(opts.AggregationPolicy) {
 		if !ValidObjectIdentifier(opts.AggregationPolicy.AggregationPolicy) {
-			errs = append(errs, errInvalidIdentifier("CreateViewOptions", "AggregationPolicy"))
-		}
-	}
-	if valueSet(opts.Columns) {
-		for i, columnOption := range opts.Columns {
-			if valueSet(columnOption.MaskingPolicy) {
-				if !ValidObjectIdentifier(columnOption.MaskingPolicy.MaskingPolicy) {
-					errs = append(errs, errInvalidIdentifier(fmt.Sprintf("CreateViewOptions.Columns[%d]", i), "MaskingPolicy"))
-				}
-			}
-			if valueSet(columnOption.ProjectionPolicy) {
-				if !ValidObjectIdentifier(columnOption.ProjectionPolicy.ProjectionPolicy) {
-					errs = append(errs, errInvalidIdentifier(fmt.Sprintf("CreateViewOptions.Columns[%d]", i), "ProjectionPolicy"))
-				}
-			}
+			errs = append(errs, ErrInvalidObjectIdentifier)
```

Comment:
Were those validations overridden by accident or all of the removed ones were not needed? (you could revert this one at least)

Reply:
Reverted.